src/parser.rs

```diff
@@ -430,6 +410,17 @@ impl<'run, 'src> Parser<'run, 'src> {
           Some(Keyword::Alias) if self.next_are(&[Identifier, Identifier, ColonEquals]) => {
             items.push(Item::Alias(self.parse_alias(attributes)?));
           }
+          Some(Keyword::Mod)
+            if self.next_are(&[Identifier, Identifier, Comment])
```

Comment:
This condition is so heinous that we should deduplicate it. Maybe `Parser::next_is_module`?